examples/weight_compression/openvino/tiny_llama/README.md

```diff
@@ -0,0 +1,34 @@
+# Weight compression of TinyLLama OpenVINO Model
+
+This example demonstrates how to find optimal ratio and group size to compress weights of the TinyLLama model from the HuggingFace Transformers framework. OpenVINO backend supports mixed precision weight compression with a 4-bit data type as a primary precision. The fastest mixed-precision mode is `INT4_SYM`, but it may lead to a significant accuracy degradation. In this example, the allowed maximum deviation from the original model is `0.2`. If the similarity of the compressed model is not satisfying, there are 2 hyper-parameters to tune: `group_size` and `ratio`. Lower group size and less ratio of 4-bit layers usually improve accuracy at the sacrifice of inference speed.
```

Comment:
```suggestion
This example demonstrates how to find the appropriate `ratio` and `group_size` parameters to compress the weights of the TinyLLama model from the HuggingFace Transformers. OpenVINO backend supports inference of mixed-precision models with weights compressed to a 4-bit data type as a primary precision. The fastest mixed-precision mode is `INT4_SYM`, but it may lead to a significant accuracy degradation, especially for models of moderate size. In this example, the allowed maximum deviation from the original model is `0.2` points of the similarity metric. If the similarity of the compressed model is not satisfying, there are 2 hyper-parameters to tune: `group_size` and `ratio`. Smaller `group_size` and `ratio` of 4-bit layers usually improve accuracy at the sacrifice of model size and inference latency.
```